src/pool/mod.rs

```diff
@@ -40,14 +47,16 @@ impl Pool {
         }
     }
 
+    /// Returns the fee percentage associated with the pool.
```

Comment:
Can we reword this to "Returns the fee associated with the pool, represented as a percentage?